### DISCUSSION TITLE
GitHub Stars - It’s time to congratulate Anoop and Bruno to the whole community!

### DISCUSSION THREAD

**Discussion by notfounnd** [usebruno/bruno#1440] [General]
https://stars.github.com/

I think this is a simple way to congrats, let's nominate him!

About the program:

> The GitHub Stars program thanks GitHub’s most influential developers and gives them a platform to showcase their work, reach more people, and shape the future of GitHub.

**Comment by helloanoop** [comment]
Thank you @notfounnd ! Hope you continue to enjoy the tool :)

I'm going to close this discussion as part of our ongoing effort to clean up open discussions, but thanks again for thinking of me!
